monai/data/meta_obj.py

```diff
@@ -189,9 +189,7 @@ def get_default_meta(self) -> dict:
 
     def __repr__(self) -> str:
         """String representation of class."""
-        out: str = super().__repr__()
```

Comment:
I think this line should be reverted, I think the output should be the same as `torch.Tensor`, but with the extra metadata (now we have metadata _instead_ of voxel data). @wyli what do you think?

Reply:
sure, please feel free to change, the corresponding test case is this line: https://github.com/Project-MONAI/MONAI/blob/18fca8811a3cc6fdbd494cf5ebe5a0d8b6392acb/tests/test_meta_tensor.py#L201